food-truck-frontend/src/pages/_document.js

```diff
@@ -1,21 +1,21 @@
 import React from 'react';
 import Document, {Head, Html, Main, NextScript} from 'next/document';
 import {ServerStyleSheets} from '@material-ui/core/styles';
-import {theme} from '../util/theme';
+// import {theme} from '../util/theme';
```

Comment:
Was this supposed to be deleted, or left commented out.

Reply:
Will delete.